ResearchKit/ActiveTasks/ORKAudioStepViewController.m

```diff
@@ -75,10 +72,16 @@ - (void)viewDidLoad {
 
 - (void)viewDidAppear:(BOOL)animated {
     [super viewDidAppear:animated];
-    
     [self start];
 }
 
+- (void)prepareStep {
+    if (_audioRecorder.outputDirectory == nil) {
```

Comment:
Errm, but you don't have the audio recorder at this point.

Check self.outputDirectory ?